CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- [#1078](https://github.com/nf-core/sarek/pull/1078) - Update tabix/bgziptabix module to fix typo
```

Comment:
```suggestion
- [#1078](https://github.com/nf-core/sarek/pull/1078) - Update tabix/bgziptabix module to fix typo
- [#1079](https://github.com/nf-core/sarek/pull/1045) - Fixed typo in profile name for tower aws megatests
```

Reply:
somehow my commit about this did not register on the other PR.
Tried to push it, not registering either, so doing it here on the UI